src/forms/button/ButtonGroup.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {RadioButton} from "../radio";
+import * as utils from "../../utils";
+import {ReactElement, ReactNode} from "react";
+import {classNames} from "../../utils";
+
+type ButtonGroupProps = {
+    className?: string;
+    children?: React.ReactNode[];
+    style?: any;
+    defaultValue?: any;
```

Comment:
What is purpose of 'defaultValue' prop ? In stories you are passing values to this as string then why it is of type 'any' ?

Reply:
Defaultvalue is to enable one button in buttongroup

Reply:
I just want to make sure that prop name is consistent with RadioButtonGroup.

Reply:
ok